package.json

```diff
@@ -64,7 +64,7 @@
     "fs-extra": "^9.0.0",
     "gh-pages": "^4.0.0",
     "istanbul": "^0.4.5",
-    "jsdom": "^15.2.0",
+    "jsdom": "^22.1.0",
```

Comment:
Is it possible to require `^20.0.0` like Jest: https://github.com/jestjs/jest/blob/main/packages/jest-environment-jsdom/package.json#L27

Reply:
It's a dev dep, so there isn't much value in leaving the range open? At runtime you have to bring your own jsdom, and from what I can tell we do support the last many major versions. Would obviously be nice to test that this is the case, but that would require a different setup/build matrix I think.

Reply:
Oh great - let's just go ahead then.